disco-categories.xml

```diff
@@ -12,6 +12,12 @@
     <title>Service Discovery Identities</title>
     &LEGALNOTICE;
     <overview>This is the official registry of values for the 'category' and 'type' attributes of the &lt;identity/&gt; element within the 'http://jabber.org/protocol/disco#info' namespace (see &xep0030;), as registered with the &REGISTRAR;.</overview>
+    <revision>
+      <version>0.28</version>
+      <date>2022-11-26</date>
```

Comment:
nit: date is in the future

Reply:
Ever heard of time travel?



OK, I fixed it ;)

Reply:
Fixed it! Thanks for reviewing :)